CODE-GUIDELINES.md

```diff
@@ -327,386 +110,290 @@ Follow this comment structure:
 }
 ```
 
-### 1. License header
+##### 1. License header
 
 PatternFly is license under the [Apache License, Version 2.0](https://www.apache.org/licenses/LICENSE-2.0).
 
 - Always add the Apache v2.0 license header at the top of each scss files.
 - Leave three blank lines bellow.
 
-### 2. DocBlock
+##### 2. Doc Block
 
 - Includes the name of the component and an optional comment.
 - Leave three blank lines bellow.
 
-### 3. Section
+##### 3. Section
 
-The Section comment is a section divider or describes a block of code.
+This comment is a section divider or describes a block of code.
 
 - Leave one blank lines above.
 
-### 4. Line
+##### 4. Line
 
 Describes a specific line of code.
 
 
-<!-- ============================================================ -->
+### Additions
 
-## Naming Selectors
+#### Lintable CSS rules
 
-Create names with useful or specific meaning via a mechanism that will not inhibit your and your team’s ability to recycle and reuse CSS.
+The CSS linter is PatternFly's single source of truth for CSS syntax, declaration order and other CSS rules like disallow type selectors, disallow vendor prefixes, and more.
 
+#### Shorthand notation
 
-- Avoid excessive and arbitrary shorthand notation, e.g., `.btn` is useful for button, but `.s` doesn't mean anything.
-- Keep classes as short and succinct as possible.
-- Use meaningful names, use structural or purposeful names over presentational.
-- Use `.js-*` classes to denote behavior (as opposed to style), but keep these classes out of your CSS.
-- Apply these same rules when creating Sass variable names.
+Limit the use of shorthand declarations to instances where you must explicitly set all the available values. Common overused shorthand properties include:
 
-```sass
-// Bad
-.t { ... }
-.red { ... }
-.header { ... }
-
-// Good
-.tweet { ... }
-.card { ... }
-.twee__header { ... }
-```
-
-### Namespace
-
-To avoid conflicts PatternFly prefixes all classes with “pf-”. This also helps differentiates PatternFly and Bootstrap class names.
+- `padding`
+- `margin`
+- `font`
+- `background`
+- `border`
+- `border-radius`
 
 ```sass
 // Bad
-.selector {
-  ...
+.element {
+  margin: 0 0 10px;
+  background: red;
+  background: url("image.jpg");
+  border-radius: 3px 3px 0 0;
 }
 
 // Good
-.pf-selector {
-  ...
+.element {
+  margin-bottom: 10px;
+  background-color: red;
+  background-image: url("image.jpg");
+  border-top-left-radius: 3px;
+  border-top-right-radius: 3px;
 }
 ```
 
-### IDs and classes
+The Mozilla Developer Network has a great article on [shorthand properties](https://developer.mozilla.org/en-US/docs/Web/CSS/Shorthand_properties) for those unfamiliar with notation and behaviour. And also [Harry Robert's](https://csswizardry.com/2016/12/css-shorthand-syntax-considered-an-anti-pattern/) article.
```

Comment:
This sounds a bit awkward, How about "The Mozilla Developer Network and Harry Robert's both have great articles on shorthand properties for those unfamiliar with notation and behaviour" Link the names.